Makefile

```diff
@@ -97,17 +67,17 @@ sast-report: $(GOSEC)
 # -------------------------------------------------------------------------
 # Run tests
 .PHONY: test-unit
-test-unit: $(GINKGO) $(GOTESTFMT)
+test-unit: $(GINKGO)
```

Comment:
Please also include `internal/utils/imagevector` package for the ginkgo unit tests